apps/avatax/src/modules/saleor/order-factory.ts

```diff
@@ -0,0 +1,22 @@
+import { SaleorOrder } from "./order";
+
+export class SaleorOrderFactory {
+  private static getData = (pricesEnteredWithTax: boolean) => ({
+    order: {
+      channel: {
+        taxConfiguration: {
+          pricesEnteredWithTax,
+        },
+        slug: "test",
+      },
+      status: "UNCONFIRMED",
+      id: "test",
+    },
+  });
+
+  public static create({ pricesEnteredWithTax }: { pricesEnteredWithTax: boolean }) {
+    const data = SaleorOrderFactory.getData(pricesEnteredWithTax);
+
+    return new SaleorOrder(data);
+  }
+}
```

Comment:
Looks like this factory is actually creating some mock. So I'd name it explicitly like this. MockFactory or something.